helix-core/src/test/java/org/apache/helix/task/TestTaskStateModelFactory.java

```diff
@@ -91,28 +93,22 @@ public void testZkClientCreationMultiZk() throws Exception {
 
     RoutingDataManager.getInstance().reset();
     RealmAwareZkClient zkClient = TaskStateModelFactory.createZkClient(anyParticipantManager);
-    Assert.assertEquals(TaskUtil
-        .getTargetThreadPoolSize(zkClient, anyParticipantManager.getClusterName(),
-            anyParticipantManager.getInstanceName()), TEST_TARGET_TASK_THREAD_POOL_SIZE);
-    Assert.assertTrue(zkClient instanceof FederatedZkClient);
+    verifyThreadPoolSizeAndZkClientClass(zkClient, anyParticipantManager,
```

Comment:
Why not also include the zkclient creation `RealmAwareZkClient zkClient = TaskStateModelFactory.createZkClient(anyParticipantManager);` in the verify method? It seems a better practice to create and close the zkclient in the same method.

Reply:
The test case is for zkClient, so style-wise I didn't want to wrap the main actor in a helper; but since it's a test case, sure, why not.